templates/workloads/partials/cf/volumes.yml

```diff
@@ -1,11 +1,9 @@
-{{- if .Storage.Volumes}}
+{{- if or .Storage.Volumes .Storage.ManagedVolumeInfo}}
 Volumes:
 {{- if .Storage.ManagedVolumeInfo}}
   - Name: {{.Storage.ManagedVolumeInfo.Name}}
     EFSVolumeConfiguration:
-      FilesystemId:
-        Fn::ImportValue:
-          !Sub '${AppName}-${EnvName}-FilesystemID'
+      FilesystemId: !GetAtt EnvControllerAction.ManagedFileSystemID
```

Comment:
I guess we haven't updated env controller for this. Is it safe for to use `!GetAtt EnvControllerAction.ManagedFileSystemID`?

Reply:
This is exported in the env stack and should be available in the Attributes of the EnvControllerAction. It's also gated behind a conditional on ManagedFileSystemInfo in the template so is only referenced when the Env Controller will have updated.